src/test/java/teammates/test/driver/TestProperties.java

```diff
@@ -133,20 +125,22 @@ public void verifyReadyForGodMode() {
         }
     }
 
-    private boolean areTestAccountsReadyForGodMode() {
-        if (!inst().TEST_STUDENT1_ACCOUNT.startsWith("alice.tmms.")) {
+    private static boolean areTestAccountsReadyForGodMode() {
+        if (!TEST_STUDENT1_ACCOUNT.startsWith("alice.tmms.")) {
             return false;
         }
-        String uniqueId = inst().TEST_STUDENT1_ACCOUNT.substring("alice.tmms.".length());
+        String uniqueId = TEST_STUDENT1_ACCOUNT.substring("alice.tmms.".length());
         if (uniqueId.isEmpty()) {
             return false;
         }
-        return inst().TEST_STUDENT2_ACCOUNT.equals("charlie.tmms." + uniqueId)
-            && inst().TEST_INSTRUCTOR_ACCOUNT.equals("teammates.coord." + uniqueId)
-            && inst().TEST_ADMIN_ACCOUNT.equals("yourGoogleId." + uniqueId);
+        
+        boolean isSecondStudentAccountReady = ("charlie.tmms." + uniqueId).equals(TEST_STUDENT2_ACCOUNT);
+        boolean isInstructorAccountReady = ("teammates.coord." + uniqueId).equals(TEST_INSTRUCTOR_ACCOUNT);
+        boolean isAdminAccountReady = ("yourGoogleId." + uniqueId).equals(TEST_ADMIN_ACCOUNT);
+        return isSecondStudentAccountReady && isInstructorAccountReady && isAdminAccountReady;
```

Comment:
<code>
return TEST_STUDENT2_ACCOUNT.equals("charlie.tmms." + uniqueId) 
            && TEST_INSTRUCTOR_ACCOUNT.equals("teammates.coord." + uniqueId) 
            && TEST_ADMIN_ACCOUNT.equals("yourGoogleId." + uniqueId);
</code>

Checkstyle will complain of `String literal expressions should be on the left side of an equals comparison.`

<code>
 return ("charlie.tmms." + uniqueId).equals(TEST_STUDENT2_ACCOUNT) && ("teammates.coord." + uniqueId).equals(TEST_INSTRUCTOR_ACCOUNT) && ("yourGoogleId." + uniqueId).equals(TEST_ADMIN_ACCOUNT);
</code>

PMD complains of `UselessParentheses`, although the parenthesis are necessary.